subprojects/docs/src/docs/userguide/custom_tasks.adoc

```diff
@@ -162,26 +184,31 @@ A task may only contain a single incremental task action.
 [[sec:which_inputs_are_considered_out_of_date]]
 === Which inputs are considered out of date?
 
-When Gradle has history of a previous task execution, and the only changes to the task execution context since that execution are to input files, then Gradle is able to determine which input files need to be reprocessed by the task. In this case, the link:{groovyDslPath}/org.gradle.api.tasks.incremental.IncrementalTaskInputs.html#org.gradle.api.tasks.incremental.IncrementalTaskInputs:outOfDate(org.gradle.api.Action)[IncrementalTaskInputs.outOfDate(org.gradle.api.Action)] action will be executed for any input file that was _added_ or _modified_, and the link:{groovyDslPath}/org.gradle.api.tasks.incremental.IncrementalTaskInputs.html#org.gradle.api.tasks.incremental.IncrementalTaskInputs:removed(org.gradle.api.Action)[IncrementalTaskInputs.removed(org.gradle.api.Action)] action will be executed for any _removed_ input file.
+When Gradle has history of a previous task execution, and the only changes to the task execution context since that execution are to incremental input file properties, then Gradle is able to determine which input files need to be reprocessed by the task.
```

Comment:
Do we define what the "task execution context" is? Can we use a better term perhaps?

Reply:
How about we say that `and the only changes to the task inputs and outputs since` instead?

Reply:
I reworded the sentence.